Modules/_hacl/include/krml/FStar_UInt_8_16_32_64.h

```diff
@@ -9,11 +9,31 @@
 
 #include <inttypes.h>
 #include <stdbool.h>
-
+#include "krml/internal/compat.h"
 #include "krml/lowstar_endianness.h"
-#include "krml/types.h"
+#include "krml/internal/types.h"
 #include "krml/internal/target.h"
 
+extern krml_checked_int_t FStar_UInt64_n;
```

Comment:
I feel like there was a big sed before that would "clean up" this file -- are you ok with the extra cruft? (I personally don't mind it, and I think it's better to make the script simple for consuming new versions of HACL*, but just want to check you're aware of this)

Reply:
I also like keeping the refresh.sh script simple where feasible.  if this causes us any problems during the 3.14beta period we can revisit.